lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -122,7 +122,7 @@ class Project
     protected $formFilterLayers = array();
 
     /**
-     * @var array
+     * @var object
```

Comment:
Why object ?

Reply:
Because it is. The content is loaded from the cfg file, with `json_decode` which returns array as objects since we don't pass the `JSON_OBJECT_AS_ARRAY` flag. And our code acts on this property as an object.

Reply:
Sorry, in fact, some code use this property as an array, and some other code use this property as an object. This is very strange :-/